aspnet/RVTR.Lodging.ObjectModel/Models/LodgingModel.cs

```diff
@@ -18,6 +18,45 @@ public class LodgingModel : IValidatableObject
 
     public IEnumerable<ReviewModel> Reviews { get; set; }
 
+    public int Bathrooms { get; set; }
+
+    public int CountAvailability ()
+    {
+      int counter = 0;
+      foreach(var rental in this.Rentals)
+      {
+        if (rental.Status == "available")
+        {
+          counter += 1;
+        }
+      }
+      return counter; 
+    }
+
+    public Dictionary<string, int> CheckLodgingTypeAvailability ()
+    {
+      Dictionary<string, int> rentalType = new Dictionary<string, int>();
+      foreach(var rental in this.Rentals)
+      {
+        if (!rentalType.ContainsKey(rental.Type))
+        {
+          rentalType.Add(rental.Type, 0);
+          if (rental.Status == "available")
+          {
+            rentalType[rental.Type] += 1;
+          } 
+        }
+        else
+        {
+          if (rental.Status == "available")
+          {
+            rentalType[rental.Type] += 1;
+          }
+        }
+      }
+      return rentalType;
+    }
+
```

Comment:
Do we need to keep track of the availability here anymore if this is in the front end